### PR TITLE
{vis}[intel/2016a] Glade v3.8.5

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2668.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/0722b7891be4e3953665 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2151.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.11
See https://gist.github.com/fef7ef0ef71ce9b03172 for a full test report.

File: easybuild/easyconfigs/i/intltool/intltool-0.51.0-intel-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+description = """intltool is a set of tools to centralize translation of
+ many different file formats using GNU gettext-compatible PO files."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+source_urls  = ['http://launchpad.net/intltool/trunk/%(version)s/+download/']
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [
+    ('XML-Parser', '2.41', '-Perl-5.20.3'),
```
Comment (wpoely86) [inline easybuild/easyconfigs/i/intltool/intltool-0.51.0-intel-2016a.eb]: perl?

Comment (boegel) [inline easybuild/easyconfigs/i/intltool/intltool-0.51.0-intel-2016a.eb]: OK, I'll let this trickly up into `versionsuffix`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2668.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/0fbe57ff000065d3fce8 for a full test report.

**Comment by wpoely86** [comment]
:+1:

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
node2668.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ab48219fc4e68e9493e9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
node2151.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e6cbacffdea9b190243b for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**